### DISCUSSION TITLE
Bad background color in my Firefox Nightly, can't figure this one

### DISCUSSION THREAD

**Discussion by gorhill** [uBlockOrigin/uAssets#22798] [Q&A] [answered]
For the longest time I noticed the default background color in my Firefox Nightly is darker than that of default profiles or that of Firefox stable.

Today I wanted to look more into why, and I find this:

![Screenshot from 2024-03-06 10-09-42](https://github.com/uBlockOrigin/uAssets/assets/585534/3f0d3929-353c-42d1-8237-b1b5537dae7e)

So there is this bad CSS color _somewhere_ missing a leading `#`. I have no idea where is that CSS declaration, I can't find a such `base.css` file anywhere -- and I can't find any `userChrome.css` or `userContent.css` anywhere in my home older.

Any of you know how to get to the bottom of this?

**Comment by ghost** [comment]
https://searchfox.org/mozilla-central/rev/6b0b8c1003504e302e243c300355ef4d61e24909/devtools/client/aboutdebugging/src/base.css#7

**Comment by gorhill** [comment]
Ok, I see the same faulty CSS color in a new profile, but the background is white (or close to it) in that new profile. I get that same dark gray background everywhere no background color is declared (including webpages), or when I uncheck those CSS declarations. I don't get this dark gray in new profile or Firefox stable. I have no idea where that dark gray is coming from.

This is what made me look into this this morning: <https://dada-lang.org/docs/dyn_tutorial/>. It's very difficult to read with that dark gray background:

![image](https://github.com/uBlockOrigin/uAssets/assets/585534/c33a44a0-7685-4b8a-b282-fbbae5c42f28)

Any other profile, either Firefox or Chromium is fine.

**Comment by gorhill** [comment, marked as answer]
Ok, found the cause, it's `browser.display.use_system_colors` in `about:config`, it was `true` (not default). Toggling back `false` fixes it. I typically never change `about:config` stuff, so I might have changed a long time ago for debugging purpose (maybe related to dark theme work) and forgot to reset it.
